lib/cylc/review.py

```diff
@@ -51,6 +51,18 @@
 from cylc.suite_srv_files_mgr import SuiteSrvFilesManager
 
 
+CYLC8_TASK_STATUSES_ORDERED = [
+    'expired',
+    'failed',
+    'preparing',
+    'running',
+    'submitted',
+    'submit-failed',
+    'succeeded',
+    'waiting',
```

Comment:
Could use string literals from task states (as the cylc7 states do) but as legacy probably not very important.

Reply:
but it would have to get these from Cylc 8?

Reply:
Sorry, I regretted this comment and should have deleted it, 'twas a fleeting thought. I meant declaring them as string literals as the cylc7 ones are in `task_state.py`. I think this is overkill and not a good use of time.